mne/viz/misc.py

```diff
@@ -809,6 +810,15 @@ def plot_filter(h, sfreq, freq=None, gain=None, title=None, color='#1f77b4',
           by squaring it.
 
         .. versionadded:: 0.18
+    plot : list | tuple | str
+        A list of the requested plots from `time`, `magnitude` and `delay`.
+        Default is to plot all three filter properties
+        ('time', 'magnitude', 'delay').
```

Comment:
... actually since I need to find something to gripe about
```suggestion
        ('time', 'magnitude', 'delay').
        
        .. versionadded:: 0.21.0
```
Same for other param below